cmds/tcz/tcz.go

```diff
@@ -87,19 +88,26 @@ func clonetree(tree string) error {
 			return nil
 		}
 		// all else gets a symlink.
-		if link, err := os.Readlink(path); err == nil {
+
+		// If the link exists
+		if link, err := os.Readlink(path[lt:]); err == nil {
+			// Confirm that it points to the same path to be assigned
 			if link == path {
 				return nil
 			}
+
+			// If it does not, return error because tcz packages are inconsistent
 			l.Printf("Symlink: need %v -> %v but %v -> %v is already there", path, path[lt:], path, link)
```

Comment:
It kind of depends on what you mean. But if you do an ls -l of a symlink, you'll see that they present it as target -> file, which is the *opposite* of how you order the args in the system call. It makes sense to present it here as it would be presented in ls -l